### PR TITLE
*: Bump kubernetes module dependencies to 1.18.x.

### PR DESCRIPTION
**Pull request by timflannagan1**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -32,53 +30,48 @@ require (
...
+	k8s.io/code-generator => k8s.io/code-generator v0.18.5-rc.0
+	k8s.io/component-base => k8s.io/component-base v0.18.3
+	k8s.io/cri-api => k8s.io/cri-api v0.18.5-rc.0
+	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.18.3
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.18.3
+	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.18.3
+	k8s.io/kube-proxy => k8s.io/kube-proxy v0.18.3
+	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.18.3
 	// Pin to kube 1.13 because of OLM
-	k8s.io/kubectl => k8s.io/kubectl v0.0.0-20191114113550-6123e1c827f7
```
Comment (bentito) [inline go.mod:74]: I think the `Pin to kube 1.13...` is/was irrelevant now.

Comment (timflannagan1) [reply go.mod:74]: Yeah, I removed that commit in one of the latter vendor commits. I don't know if I have the heart to go back and remove it from the first commit.

**Comment by timflannagan1** [comment]
Overriding e2e aws test due to a known bug:

```
time="2020-06-19T18:51:59Z" level=error msg="error collecting metrics" app=metering chunkSize=5m0s component=PrometheusImporter endTime="2020-06-19 18:51:59.711787596 +0000 UTC" error="failed to perform Prometheus query: execution: found duplicate series for the match group {namespace=\"metering-prometheusconnectorworks\", persistentvolumeclaim=\"hdfs-namenode-data-hdfs-namenode-0\"} on the right hand-side of the operation: [{namespace=\"metering-prometheusconnectorworks\", persistentvolumeclaim=\"hdfs-namenode-data-hdfs-namenode-0\", storageclass=\"gp2\", volumename=\"pvc-087775fc-25e1-41af-b991-f36200e9af27\"}, {namespace=\"metering-prometheusconnectorworks\", persistentvolumeclaim=\"hdfs-namenode-data-hdfs-namenode-0\", storageclass=\"gp2\"}];many-to-many matching not allowed: matching labels must be unique on one side" logID=WoPk2WNXyJ namespace=metering-validhdfs-reportdynamicinputdata reportDataSource=persistentvolumeclaim-request-bytes startTime="2020-06-19 18:42:00 +0000 UTC" stepSize=1m0s tableName=hive.metering.datasource_metering_validhdfs_reportdynamicinputdata_persistentvolumeclaim_request_bytes
```

/override ci/prow/metering-e2e-aws

**Pull request merged**